dll/shellext/fontext/CFontExt.cpp

```diff
@@ -589,7 +586,8 @@ HRESULT CFontExt::DoInstallFontFile(LPCWSTR pszFontPath, LPCWSTR pszFontsDir, HK
     }
 
     DWORD cbData = (wcslen(pszFileTitle) + 1) * sizeof(WCHAR);
-    LONG nError = RegSetValueExW(hkeyFonts, szFontName, 0, REG_SZ, (const BYTE *)szFontName, cbData);
+    LONG nError = RegSetValueExW(hkeyFonts, strFontName, 0, REG_SZ,
+                                 (const BYTE *)(LPCWSTR)strFontName, cbData);
```

Comment:
Why is the length calculated on something else than the actual string that is stored here?

Reply:
probably the argument to be written should be `pszFileTitle` ?

Reply:
Fixed.